flags-sdk/hypertune/README.md

```diff
@@ -0,0 +1,42 @@
+# Flags SDK Template
+
+This example uses a template adapter for feature flags with the [Flags SDK](https://flags-sdk.dev) that works with the [Flags Explorer](https://vercel.com/docs/workflow-collaboration/feature-flags/using-vercel-toolbar).
+
+## Demo
+
+[https://hypertune-flags-sdk.vercel.app/](https://hypertune-flags-sdk.vercel.app/)
+
+## How it works
+
+This demo shows different combinations of two banners on the home page.
+The banners are configured to show/hide each banner 50% of the time.
+
+Once you visit the page, you can see a variation of both/one/none of the banners.
+Since this example is using a stable id to identify users, you will see the same variation until you reset your id.
+
+To test different variations, you can use the Dev Tools at the bottom to reset the stable id and reload the page.
+This allows you to test different variations of the banners.
+
+If you clone the example, you can also use the [Flags Explorer](https://vercel.com/docs/workflow-collaboration/feature-flags/using-vercel-toolbar) to test different variations by creating overrides.
```

Comment:
The latest link to this is https://vercel.com/docs/feature-flags/flags-explorer, as the old link redirects to that page now